config/main.py

```diff
@@ -540,7 +550,8 @@ def load_minigraph():
         command = "{} -H -m --write-to-db".format(SONIC_CFGGEN_PATH)
     run_command(command, display_cmd=True)
     client.set(config_db.INIT_INDICATOR, 1)
-    run_command('pfcwd start_default', display_cmd=True)
+    if device_type != 'MgmtToRRouter':
```

Comment:
Isn't it better to check if device_type contains 'Mgmt' ?

Reply:
as per our discussion, leaving this as is